commands/project/deploy.ts

```diff
@@ -132,24 +144,21 @@ exports.handler = async options => {
 
     const { data: deployResp } = await deployProject(
       derivedAccountId,
-      projectName,
+      projectName!,
       buildIdToDeploy,
       useV3Api(projectConfig?.platformVersion)
     );
 
-    if (!deployResp || deployResp.error) {
-      logger.error(
-        i18n(`${i18nKey}.errors.deploy`, {
-          details: deployResp.error.message,
-        })
-      );
+    // TODO: Validate this change. Were we actually using deployResp.error.message?
+    if (!deployResp) {
+      logger.error(i18n(`${i18nKey}.errors.deploy`));
       return process.exit(EXIT_CODES.ERROR);
     }
 
     await pollDeployStatus(
       derivedAccountId,
-      projectName,
-      deployResp.id,
+      projectName!,
+      Number(deployResp.id),
```

Comment:
I would double check what type this actually is - it might be mistyped in LDL, or we might want to update `pollDeployStatus` to accept a string

Reply:
I double checked this in the api catalog and the api does in fact return the id as a string. I could update the polling functions to take it in as a string, but it looks like it's actually more convenient to treat them as numbers in that function ([here](https://github.com/HubSpot/hubspot-cli/blob/main/lib/projects/buildAndDeploy.ts#L152), for example).